doc improvement : update figure of led states

divipillai opened the pull request:
Update the nrf_cloud_led_states figure.
The figure is displayed in the
nRF9160: LTE Sensor Gateway sample

Signed-off-by: divya pillai <divya.pillai@nordicsemi.no>

greg-fer reviewed (changes requested):
Some comments:

- I don't think the green is Nordic Grass. I would rather use one of the primary colors for the On state.
- I would use blocks for states shown with text ("Startup", "LTE connecting"), also because this is inconsistent use of text with the text "Not associated" and "Add device on nRF Cloud".
- Text should not be on top of the arrow shape, but next to it. See for example asset_tracker_v2_active_state.vsdx
- For the legend, use text next to the color, and use transparent background in the legend, so that it doesn't seem part of the diagram. For example, see caf_sample_structure.vsdx

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-8611
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

NordicBuilder commented:
### Integration test specification

| Test Module | File based changes | Manually selected | West overwrite |
| ----------- | :----------------: | :---------------: | :------------: |

Test spec set is empty, this is probabley due to no matching entry in [.github/test-spec.yml](https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml)


[Detailed information of selected test modules](https://jenkins-ncs.nordicsemi.no/job/latest/job/sub/job/test-fw-nrfconnect-nrf/job/master/26533/artifact/test-spec/test-spec-summary.yml/*view*/)

*Note: This message is automatically posted and updated by the CI*

divipillai commented:
> Some comments:
> 
> * I don't think the green is Nordic Grass. I would rather use one of the primary colors for the On state.
> * I would use blocks for states shown with text ("Startup", "LTE connecting"), also because this is inconsistent use of text with the text "Not associated" and "Add device on nRF Cloud".
> * Text should not be on top of the arrow shape, but next to it. See for example asset_tracker_v2_active_state.vsdx
> * For the legend, use text next to the color, and use transparent background in the legend, so that it doesn't seem part of the diagram. For example, see caf_sample_structure.vsdx

1. I have used Nordic Grass. Could you please check again.
2. Updated
3. Updated
4. Updated
